src/transaction.ts

```diff
@@ -0,0 +1,234 @@
+import { Psbt, networks } from 'liquidjs-lib';
+import { estimateTxSize, UtxoInterface } from './wallet';
+
+export interface OutputInterface {
```

Comment:
```suggestion
export interface RecipientInterface {
```
I think it's better to not conflate with tx outputs.